play/server-circe/src/main/scala/endpoints/play/server/circe/JsonEntities.scala

```diff
@@ -22,10 +22,10 @@ trait JsonEntities extends Endpoints with algebra.JsonEntities {
   type JsonResponse[A] = CirceEncoder[A]
 
   def jsonRequest[A : CirceDecoder]: RequestEntity[A] =
-    playComponents.playBodyParsers.raw.validate { buffer =>
-      jawn.parseFile(buffer.asFile)
+    playComponents.playBodyParsers.tolerantText.validate { text =>
+      parser.parse(text)
         .right.flatMap(CirceDecoder[A].decodeJson)
-        .left.map(error => Results.BadRequest)
+        .left.map(ignoredError => Results.BadRequest)
     }
 
   def jsonResponse[A : CirceEncoder]: Response[A] = a => Results.Ok(CirceEncoder[A].apply(a))
```

Comment:
Why do we want to have direct circe support for play-server but dropped all the others? Just asking.

Reply:
I find this one useful when I directly write a server, without having a “shared” description of the HTTP API. In such a case I don’t even need a JSON decoder for responses (only an encoder), so I don’t use the `JsonEntitiesFromCodec` thing.

Reply:
I see. It may be a little confusing that only a single interpreter has direct support, but maybe that's not a problem :)